### PR TITLE
#109 App version update

### PR DESCRIPTION
**Pull request by lijogeorgep**
update app version to 2023.10.1+16

### CODE REVIEW

**Comment by lalc** [comment]
Why 1.10.1?

**Comment by lijogeorgep** [comment]
1st year 10th month(ie:october) 1st week . (1.10.1)

**Comment by lijogeorgep** [comment]
Am I need to change this ?

**Comment by josmilan** [comment]
As our release plan, we follow this method for our version naming.

**Comment by lalc** [comment]
Since we are in 2023, it should be 3.10.1? 😊

Or 2023.10.1 @georgepadayatti?

**Comment by josmilan** [comment]
In Data4diabetes we started this year only. That's why our version starts with "1." in the data wallet this is 3rd year. That's why its "3." 

Do we need to update it to "3."

**Comment by lalc** [comment]
Well. That would be hard to follow and manage. This is why we want to align with the year.

**Comment by lijogeorgep** [comment]
@lalc @josmilan Then Can I update the version as 3.10.1?

**Comment by lalc** [comment]
> @lalc @josmilan Then Can I update the version as 3.10.1?

@georgepadayatti @josmilan Please decide the best option.

**Review by georgepadayatti — APPROVED** [review]
